src/hooks/useSegmentAnalytics.js

```diff
@@ -26,7 +26,7 @@ export default function useSegmentAnalytics() {
         setPrevPath(currentPath);
         // NOTE (appleseed): if analytics aren't showing the full pathname + location.hash then we need to manually pass it
         // ... into analytics.page() below
-        analytics.page();
+        analytics.page(currentPath);
```

Comment:
Last thing. This wasn't what I expected. So I just want to make sure you've verified this:

[The `analytics.page` docs](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/#page) make it look like this call should be `analytics.page("PageName", {path: currentPath})` where PageName could just be like "App". And maybe in Olympus Pro it could be "Pro". Something like that. Maybe your implementation above just works?

Reply:
If only one arg is passed in, it behaves as follows: 
_Note: if you pass only one string to page it is assumed to be name. 

Currently the data being pushed in will say something like /bonds, /stake, and so on, not differentiating what app it came from... but we also have different keys for olympus pro, the app and so on, so it should be okay leaving it as such

Reply:
alright if you're good with it I'm good with it. Thanks for checking!